CONTRIBUTING.md

```diff
@@ -103,13 +103,39 @@ cases this would be the component name. For example select, button, etc.
 
 ### Message
 
+**Subject**  
 The commit message should describe the problem it solves or the feature it
 introduces. Not the changes you have done. Furthermore the commit message has to
 start with an uppercase letter and ends with a stop.
 
-### Example
+**Body**  
+The body should include the motivation for the change and contrast this with
+previous behavior.
 
-`feat(filter-field): Added node removal on backspace.`
+**Footer**  
+The footer should contain any information about Breaking Changes and is also the
+place to reference GitHub issues that this commit Closes.
+
+Breaking Changes should start with the word BREAKING CHANGE: with a space or two
+newlines. The rest of the commit message is then used for this.
+
+### Examples
+
+```
+feat(filter-field): Added node removal on backspace.
+```
+
+```
+fix(graphite): Atop graphite breaking when width < 0.1.
```

Comment:
You could provide a sensible example, that is related to the component library.
Even though if you feel like an artist that loves to sketch with some graphite ✏️.